app/assets/javascripts/models/page/search_page.js.coffee

```diff
@@ -31,12 +33,14 @@ ns.SearchPage = do (ko
                     PreferencesModel = data.Preferences
                     FeedbackModel = ui.Feedback
                     SiteTourModel = ui.SiteTour
+                    ReverbRetirementModel = ui.ReverbRetirement
                     url = @edsc.util.url
                     StateManager = ui.StateManager) ->
   current = null
 
   preferences = new PreferencesModel()
-  sitetour = new SiteTourModel();
+  sitetour = new SiteTourModel()
+  reverbRetirement = new ReverbRetirementModel()
```

Comment:
this shouldn't be a new class and don't initialize it here.

Reply:
You're right - doesn't need it after all!  Done in next push.